web/webkit/src/main/scala/net/liftweb/http/LiftSession.scala

```diff
@@ -2325,32 +2326,68 @@ class LiftSession(private[http] val _contextPath: String, val underlyingId: Stri
    * the CometActor is instantiated.  If the CometActor already exists
    * in the session, the message will be sent immediately.  If the CometActor
    * is not yet instantiated, the message will be sent to the CometActor
-   * as part of setup (@see setupComet) if it is created as part
+   * as part of setup (@see queueCometMessage) if it is created as part
    * of the current HTTP request/response cycle.
    *
    * @param theType the type of the CometActor
+   * @param msg the message to send to the CometActor
+   */
+  def sendCometActorMessage(theType: String, msg: Any) {
```

Comment:
It's [`findOrCreateComet[T]`](https://github.com/lift/framework/blob/master/web/webkit/src/main/scala/net/liftweb/http/S.scala#L844-L870), sorry.